applications/tari_dan_wallet_web_ui/src/routes/AssetVault/Components/SendMoney.tsx

```diff
@@ -189,7 +189,10 @@ export function SendMoneyDialog(props: SendMoneyDialogProps) {
             });
             return;
           }
-          setTransferFormState({ ...transferFormState, fee: result.fee.toString() });
+          // temporary fix for having a proper calculation for estimated fee to avoid not enough fee issue
+          // TODO: remove once fixed in backend
```

Comment:
This is a bit misleading, it's not a backend bug to be fixed _per se_. The fee calculations are working as intended; the difference in fees is a consequence of the dry run and "normal" transactions differing in an expected way causing the fees to differ.
A suggestion to rectify this is to modify the fee calculation to charge a flat fee for withdrawal proof type. Another solution is for the wallet to reuse the rangeproof verbatim (but this may not be possible/desireable when we include the dry run flag in the transaction. See #1376 . However, this is a much simpler solution to this pesky annoyance. 

```suggestion
          // Simple fix for the estimated fee differing between the dry-run and non-dry-run transactions.
          // Since fees are charged for the transaction byte size and for confidential transfers, the rangeproof 
          // may differ in length and, therefore in fees. The fees may differ typically by 2/3, this more than 
          // accounts for that. See https://github.com/tari-project/tari-dan/issues/1312
          // TODO: remove once this is no longer an issue
```